test/unit/components/info-button.test.jsx

```diff
@@ -26,7 +35,7 @@ describe('InfoButton', () => {
                 message="Here is some info about something!"
             />
         );
-        component.find('div.info-button').simulate('click');
+        component.find('div.info-button').simulate('mouseOver');
```

Comment:
why did you change this part of the test?  Looks like the message is still expected to be shown on click, right?

Reply:
Yes, it should be shown on either `click` or `mouseOver`. This test is focused on the behavior of `mouseOut`, and I thought `mouseOver` would be a more common step before `mouseOut` than `click`.

Reply:
I was just looking at the diff wrong (missed the part that it collapsed and thought this was in the test after). Sorry about that.